src/components/EditProfileModal.tsx

```diff
@@ -141,11 +167,33 @@ export const EditProfileModal = ({
         }}
       >
         <Text h2>Edit Profile</Text>
-        <Text p css={sectionHeader}>
-          Profile Image
-        </Text>
-        <AvatarUpload original={myProfile.avatar} />
-
+        <Flex css={{ columnGap: '$lg', '@sm': { flexDirection: 'column' } }}>
+          <Flex column css={{ alignItems: 'center' }}>
+            <Text p css={sectionHeader}>
+              Profile Image
+            </Text>
+            <AvatarUpload original={myProfile.avatar} />
+          </Flex>
+          <Flex column css={{ alignItems: 'center' }}>
+            <Text p css={sectionHeader}>
+              Profile Name{' '}
+              <Tooltip
+                content={
+                  <div>You can&apos;t edit your name during active epochs</div>
```

Comment:
Is there consensus around this? What if you are ALWAYS in an active recurring epoch?

Reply:
Seems like this warning should only show during an active epoch

Reply:
Is this *also* enforced on the server side?

Reply:
@sshmm did you see these?

Reply:
👍